recipes/dukpy/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "dukpy" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/amol-/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 081eda6d60053dc6b141171ff6cfd5dfd037ab06d4ad5fbeaf01b850029cac3d
+
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
Do you need the ``c`` compiler here?

Reply:
Yes, in the `src/` folder there are C files that get compiled into the `dukpy._dukpy` extension.